index.html

```diff
@@ -28,7 +28,7 @@ <h1>Dr.Norman Borlaug</h1>
       <figure id="img-div" class="">
```

Comment:
Please do not leave empty space in class attribute. It is not a good habit to leave empty attributes